### ISSUE TITLE
Error occurred terminating FFmpeg process

### ISSUE THREAD

**Issue by rbswift** [Sunoo/homebridge-camera-ffmpeg#748]
**Describe The Problem:**

FFmpeg keeps running after stream stops chewing up CPU
**To Reproduce:**

Open a stream to Home app on mac then close it. ffmpeg continues running 
**Logs:**

```
[8/25/2020, 2:39:33 PM] [Camera FFmpeg] [Back] Device appears to be inactive. Stopping stream.
[8/25/2020, 2:39:33 PM] [Camera FFmpeg] [Back] Error occurred terminating FFmpeg process: Error [ERR_SOCKET_DGRAM_NOT_RUNNING]: Not running
```

**Homebridge Config:**

```json
                {
                    "name": "Back",
                    "manufacturer": "Hikvision",
                    "model": "DS-2CD3132F-IW",
                    "serialNumber": "DS-2CD3132F-xxx",
                    "firmwareRevision": "V5.4.5 build 170401",
                    "videoConfig": {
                        "source": "-rtsp_transport tcp -vcodec h264_mmal -re -i rtsp://admin:frodO364@192.168.188.38/streaming/channels/2",
                        "stillImageSource": "-i http://admin:xxx@192.168.188.38/Streaming/Channels/1/picture",
                        "videoFilter": "scale=1024:576"
                    }
                },
```

**Screenshots:**

**Environment:**

* **Node.js Version**: v12.18.3
* **NPM Version**: 6.14.4
* **Homebridge Version**: 1.1.2
* **Homebridge Camera FFmpeg Version**:homebridge-camera-ffmpeg v3.0.0
* **Homebridge Config UI X Plugin Version**:homebridge-config-ui-x v4.26.0
* **Operating System**:  Raspbian buster Linux pi4 5.4.51-v7l+ #1333 SMP Mon Aug 10 16:51:40 BST 2020 armv7l GNU/Linux

**Comment by Sunoo** [comment]
Okay, I think I see the issue. It's going to have to wait until tomorrow for me to figure out the best way to fix it though. It looks like it should only happen on a camera timeout though.

When you say "close it" did you shut the lid on your Mac or something? If you close the stream in the Home app, it should do a graceful termination.

**Comment by rbswift** [comment]
Just close the live feed by clicking the X inside the app (didn't close the lid). ffmpeg seems to continue running after the home app is quit too. Haven't seen if it eventually times out yet. No rush. Cheers

**Comment by Sunoo** [comment]
Yea, this was a problem I had fixed in the past, unfortunately it will keep running forever until you `sudo killall -9 ffmpeg` or something. I'm pretty sure I found the bug, it snuck in when I reworked a big chunk of the timeout handling code in preparation for 3.0.0.

I'm just very surprised that the Home app isn't doing a graceful shutdown of the stream.

**Comment by Sunoo** [comment]
Fixed in 3.0.1.
